CHANGELOG.md

```diff
@@ -100,6 +100,7 @@ All notable changes to TTT2 will be documented here. Inspired by [keep a changel
 - Fixed draw.Arc when `gmod_mcore_test` is set to 1 (by @WardenPotato)
 - Fixed weapon help box width for wide bindings with short descriptions (by @TimGoll)
 - Fixed `GM:TTTBodySearchPopulate` using the wrong data variable (by @TimGoll)
+- Fixed the way fonts are initialized in TTT2 meaning that it is no longer broken and also works on linux (by @EntranceJew)
```

Comment:
This is misleading, remove the linux part. Because Tahoma is not a font you will find on Linux (needs to be extracted from windows isos or has to be downloaded from some other 3rd party projects).

Maybe add a note to linux users that they should install Tahoma for correct fonts

Reply:
could you please modify it so that it represents your situation on linux because what I wrote is what I assumed this did on linux. And I don't have a linux installation on hand to test it. Thanks!

Reply:
Done I think, if you 2ant something else, please just change it yourself